tests/ImageSharp.Tests/Formats/Gif/GifEncoderTests.cs

```diff
@@ -150,7 +150,7 @@ public void NonMutatingEncodePreservesPaletteCount()
                 outStream.Position = 0;
                 var clone = Image.Load<Rgba32>(outStream);
 
-                GifMetadata cloneMetaData = clone.Metadata.GetFormatMetadata<GifMetadata>(GifFormat.Instance);
+                GifMetadata cloneMetaData = clone.Metadata.GetFormatMetadata(GifFormat.Instance);
```

Comment:
Maybe better to use `var cloneMetaData` here instead?

Reply:
do u want another pr that makes the use of `var` consistent?

Reply:
That would be nice, but you can probably sneak this one in?

Reply:
I think, we should merge it as-is now to keep the ball rolling.

Reply:
@dlemstra @antonfirsov @JimBobSquarePants so re var. whats your preferences? "just var everywhere"?

Reply:
The ide should hint via editorconfig. Var only when the type is apparent.

Reply:
weird. i dont get a hint?

Reply:
Very weird, what’s your dev setup? Any plugins etc?

Reply:
I will troubleshoot when i get home, and get back to you